### ISSUE TITLE
No image support?

### ISSUE THREAD

**Issue by chris-zettelkasten** [glushchenko/fsnotes#86]
Cannot view img in preview - I tried it with the given markdown code as it works e.g. in nvALT. Bug?

**Comment by glushchenko** [comment]
Can you provide code example? Example below works for me:

```
![](https://raw.githubusercontent.com/glushchenko/fsnotes/master/code.png)
```

**Comment by chris-zettelkasten** [comment]
![](xyz.*) did not work for me.

**Comment by glushchenko** [comment]
What is xyz.*? Remote or local path? Did you use scheme prefix?

**Comment by chris-zettelkasten** [comment]
xyz.* means a filename and extension, like image.jpg or image.png or whatever.
The image file is in the same folder like the text file.
Sorry, I don't know what scheme prefix means.

**Comment by glushchenko** [comment]
Thank you for clarification, images preview from file system will be available in next releases. I working on it now. Coming soon.

**Comment by glushchenko** [comment]
Released in 1.2.3
